x/inflation/spec/01_concepts.md

```diff
@@ -52,20 +52,17 @@ The inflation distribution for staking, usage incentives and community pool is
 implemented through an exponential formula, a.k.a. the Half Life.
 
 Inflation is minted in daily epochs. During a period of 365 epochs (one year), a
-daily provision of Evmos tokens is minted and allocated to staking rewards,
-usage incentives and the community pool (`epochProvison`). The epoch provision
-doesn’t change within a period and its cumulated amount per period is equal
-total inflation per period (`epochProvision * epochsPerPeriod =
-periodProvision`).
+daily provision (`epochProvison`) of Evmos tokens is minted and allocated to staking rewards,
+usage incentives and the community pool. The epoch provision depends of parameters and calculated every day.
 
-At the end of each period , the epoch provision is recalculated with the
-calculation below:
+At the end of each epoch, the epoch provision is recalculated with the calculation below:
 
 ```latex
 periodProvision = exponentialDecay       *  bondingIncentive
 f(x)            = (a * (1 - r) ^ x + c)  *  (1 + maxVariance * (1 - bondedRatio / bondingTarget))
 
-epochProvision = periodProvision / epochsPerPeriod
+
+epochProvision = periodProvision / epochsPerPeriod * powerReduction
```

Comment:
nit: might be a good idea to include parenthesis here to make order of operations more obvious than just reading left to right, ex. `epochProvision = periodProvision / (epochsPerPeriod * powerReduction)` versus `epochProvision = (periodProvision / epochsPerPeriod) * powerReduction`.

Reply:
What is the `powerReduction`? It should definitely be explained and defined if it's added to the calculation

Reply:
I decided to not include `PowerReduction` as It can be confusing for readers(as It serves for specific purpose, and not affect inflation distribution etc), It's just 10**18(decimals) number to convert portion value to `aevmos` units (remark if I'm wrong @fedekunze), not sure why It is called `PowerReduction`, but still not necessary to include this in the docs

Reply:
> [...], but still not necessary to include this in the docs

I don't mean to cause a fuzz about something small but I strongly disagree - if it's listed it the formula, it must be explained. If it's not explained and not meaningful to the understanding then remove it from the docs.